docs/examples/add-to-your-application/go/models.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+type Repository struct {
+	Id       int
+	Name     string
+	IsPublic bool
+}
+
+var reposDb = map[string]Repository{
+	"gmail": {Id: 0, Name: "gmail"},
+	"react": {Id: 1, Name: "react", IsPublic: true},
+	"oso":   {Id: 2, Name: "oso"},
+}
+
+func GetRepositoryByName(name string) Repository {
+	return reposDb[name]
+}
+
+// docs: start
+type Role struct {
+	Name   string
+	Repository Repository
+}
+
+type User struct {
+	Roles []Role
+}
+
+var usersDb = map[string]User{
+    "larry":  {Roles: []Role{{Name: "admin", Repository: reposDb["gmail"]}}},
```

Comment:
Any reason to exclude other the other two examples here?

Reply:
I have been excluding them in all the add to your apps - I guess not for any great reason, I'm not sure they add much in the context of the page though, and users won't run the add to your app code or experiment with it.